sql/2021/compression/lighthouse_compression_byte_savings.sql

```diff
@@ -0,0 +1,17 @@
+#standardSQL
+# lighthouse_compression_byte_savings.sql :  Text Based Compression Byte Savings
+SELECT
+  _TABLE_SUFFIX AS client,
+  ROUND(CAST(JSON_EXTRACT_SCALAR(report, "$.audits.uses-text-compression.details.overallSavingsBytes") AS INT64) / 1024 / 1024) AS mbyte_savings,
+  COUNT(0) AS num_pages,
+  ROUND(COUNT(0) * 100 / SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX), 2) AS pct_pages
```

Comment:
Remove percent formatting, do in Sheets instead
```suggestion
  COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS pct_pages
```